src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -209,6 +209,10 @@ uct_rc_mlx5_iface_common_poll_rx(uct_rc_mlx5_iface_common_t *mlx5_common_iface,
 done:
     max_batch = rc_iface->super.config.rx_max_batch;
     if (rc_iface->rx.srq.available >= max_batch) {
+        if (rc_iface->rx.srq.reserved) {
```

Comment:
can we move this inside uct_rc_mlx5_iface_srq_post_recv? so slow-path code will not be inlined